gulpfile.js

```diff
@@ -218,33 +239,37 @@ gulp.task('build-frontend-games-html', ['clean-frontend'], function () {
 
 });
 
-gulp.task('build-frontend-js-vendor', ['clean-frontend'], function () {
-  
-  var b = browserify({
-    entries: buildConfig.frontend.js.vendor.src,
-    debug: true
-  });
+function buildBrowserifyBundle(bundleOptions) {
 
-  return b.bundle()
-    .pipe(source('vendor.js'))
-    .pipe(buffer())
-    .pipe(sourcemaps.init({loadMaps: true}))
-//      .pipe(uglify())
-//      .on('error', util.log)
-    .pipe(sourcemaps.write('./'))
-    .pipe(gulp.dest(buildConfig.frontend.js.vendor.dest));
+    var b = browserify({
+      entries: bundleOptions.entries,
+      debug: true
+    });
 
-});
+    b.require(bundleOptions.require);
```

Comment:
@greybubble This exposes the packages, or files, listed in the `bundleOptions.require` so that other script files can access them.  This means we can make multiple bundles that reference each other.

`vendor.js` will be where we put 3rd party scripts that are used throughout the games.